scania-digital-design-system/sdds
# Prettier and ESlint on the root level

**Describe pull-request**  

- Adding Prettier and ESlint to the SDDS level
- Adding husky and precomit to SDDS level too, removing one from components level

**Solving issue**  

- This is improving overall code quality

**How to test**  

- Run code locally and see if you can commit as usual and if your local runs smooth


**Additional context**  

- There are a lot of new packages so I would suggest running `npm run clean` from root and then `npm i` to have clean setup.
